link-git-protocol/Cargo.toml

```diff
@@ -25,7 +25,20 @@ versions = "3.0.2"
 
 [dependencies.git-repository]
 version = "0.9.0"
-features = [ "async-network-client", "unstable" ]
+default-features = false # leave out max-performance as it doesn't build on msvc win due to sha1-asm. See https://github.com/RustCrypto/asm-hashes/issues/17 for details
+features = [ "one-stop-shop", "async-network-client", "unstable" ]
+
+[dependencies.git-features]
```

Comment:
Why are we adding these back rather than accessing them through `git-repository`?

Reply:
It adds all of the default features back except for `max-performance`, which unfortunately breaks the build.

I was thinking that it might be better to remove `max-performance` from the default feature set for now, but felt bad about it.

However, now that I wrote this I revisited the issue and fixed it on `git-repository` side as the superior choice :).

Reply:
I'm confused, too. It seems like we need `git-repository` and `git-packetline`, but not the rest. Maybe a rebase went wrong or something?

Reply:
Great catch! And even though I don't know how that happened, removing the superfluous packages did indeed not cause any trouble.